go/mysql/schema.go

```diff
@@ -32,9 +32,11 @@ import (
 
 const (
 	// BaseShowPrimary is the base query for fetching primary key info.
-	BaseShowPrimary = "SELECT table_name, column_name FROM information_schema.key_column_usage WHERE table_schema=database() AND constraint_name='PRIMARY' ORDER BY table_name, ordinal_position"
-	// BaseShowTableUniqueKey returns names of colunms covered by a given unique constraint on a given table, in key order
-	BaseShowTableUniqueKey = "SELECT column_name as column_name FROM information_schema.key_column_usage WHERE table_schema=database() AND table_name=%a AND constraint_name=%a ORDER BY ordinal_position"
+	BaseShowPrimary = `SELECT table_name as table_name, COLUMN_NAME as column_name
```

Comment:
Nit: I would remove the column alias when it matches the column name to shorten and simplify the query (is it just that we want it in lower case (for the map) whereas the schema uses upper case?). I'd also make the case usage consistent. It can matter depending on the collation used and case sensitivity of the system.

Reply:
The column alias to use lower case has been used in a  lot of places. We use the column name as an alias when we convert the query result to a map with the column name as the key using the `.Named()` mechanism. MySQL can return lower or upper case depending on how it is configured. This caused a difference in case between the code and what MySQL returned. Hence the explicit aliasing. 

I agree we should change this. One idea was to add both upper and lower cases to the map so that both are found, but that would also be a hack which wouldn't work if someone used camel case, for example!